### PR TITLE
Add @kiwicom/monorepo to make graphql work with monorepo packages

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -78,7 +78,7 @@
     semver "^5.4.1"
     source-map "^0.5.0"
 
-"@babel/core@^7.0.0", "@babel/core@^7.1.0", "@babel/core@^7.1.6", "@babel/core@^7.2.0", "@babel/core@^7.3.4":
+"@babel/core@^7.0.0", "@babel/core@^7.1.0", "@babel/core@^7.1.6", "@babel/core@^7.2.0":
```
Comment (JoseDuSV) [inline yarn.lock]: Thanks, looks good. The only thing is, that after running `yarn install` I'm getting 12 mostly Babel related differences in `yarn.lock`. Looks like the problem we faced before.

Screenshot 2019-04-03 at 10 12 01

Comment (tbergq) [reply yarn.lock]: Can you try again? Should be fixed now

Comment (JoseDuSV) [reply yarn.lock]: Yes, it's ok now. 🚀

Comment (tbergq) [reply yarn.lock]: 👍 I will merge it when CI finishes

Comment (JoseDuSV) [reply yarn.lock]: Definitely, thanks.

**Pull request merged**